x-pack/plugins/maps/public/classes/layers/create_tile_map_layer_descriptor.ts

```diff
@@ -59,9 +60,18 @@ export function createAggDescriptor(
   });
   const aggType = aggTypeKey ? AGG_TYPE[aggTypeKey as keyof typeof AGG_TYPE] : undefined;
 
-  return aggType && metricFieldName && (!isHeatmap(mapType) || isMetricCountable(aggType))
-    ? { type: aggType, field: metricFieldName }
-    : { type: AGG_TYPE.COUNT };
+  if (
+    !aggType ||
+    aggType === AGG_TYPE.COUNT ||
+    !metricFieldName ||
+    (isHeatmap(mapType) && !isMetricCountable(aggType))
+  ) {
+    return { type: AGG_TYPE.COUNT };
+  }
+
+  return aggType === AGG_TYPE.PERCENTILE
```

Comment:
Is this needed? Coordinate map visualization does not support percentile metric.

<img width="300" alt="Screen Shot 2020-12-09 at 3 45 39 PM" src="https://user-images.githubusercontent.com/373691/101697910-de2b7600-3a35-11eb-908d-523f2d4ca07f.png">

Reply:
discussed offline, let's keep as is